source/read/retrieve.txt

```diff
@@ -180,7 +180,12 @@ you can set in the array:
      - Description
 
    * - ``batchSize`` 
-     - | The number of documents to return per batch. The default value is ``101``.
+     - | The maximum number of documents within each batch returned in a query result. By default,
+         the ``find()`` method has an initial batch size of ``101`` documents
```

Comment:
Since this section is talking about both `find()` and `findOne()`, I think this can just refer to "the `find` command" (used by both helpers).

Therefore, this snippet will just duplicate the first paragraph from the `batchSize` docs in `MongoDBCollection-find.txt`.